### PR TITLE
add vim and emacs in the new editors section

### PR DESCRIPTION
**Pull request by kidd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,12 +158,15 @@
...
+* [RethinkDB](https://www.reddit.com/r/rethinkdb/)
 * [SQLServer](https://www.reddit.com/r/SQLServer/) - Microsoft SQL Server Administration and T-SQL Programming.
 
+## Editors
 
+* [emacs](https://www.reddit.com/r/emacs)
+* [vim](https://www.reddit.com/r/vim)
 
-### Entertainment
+## Entertainment
```
Comment (iCHAIT) [inline README.md:169]: Why did you remove a hash?

Comment (kidd) [reply README.md:169]: because in the table of contents it appears as a "top level" entry. I thought it was a typo. 

Should I put it back to 3?

Comment (iCHAIT) [reply README.md:169]: Oh yeah, you are right! It's ok. Merging 👍
